_posts/2015-09-08-stylish-portfolio.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title:  "Stylish Portfolio"
+date:   2015-09-08 01:03:00
+homepage: https://github.com/volny/stylish-portfolio-jekyll
+download: https://github.com/volny/stylish-portfolio-jekyll/archive/master.zip
+demo: https://volny.github.io/stylish-portfolio-jekyll/
+author: Start Bootstrap
+thumbnail: stylish.png
```

Comment:
The thumbnail filename should be consistent with the post filename, but without date prefix: `stylish-portfolio.png`.